### PR TITLE
Tm 6597 edit cdo

### PR DESCRIPTION
**Pull request by RamelHenderson**

### CODE REVIEW

File: src/Components/BidderPortfolio/BidderPortfolioContainer/BidderPortfolioContainer.jsx [hunk 1]
```diff
@@ -23,30 +23,49 @@ class BidderPortfolioContainer extends Component {
   };
 
   render() {
-    const { bidderPortfolio, bidderPortfolioExtraData, pageSize, showListView, isLoading, viewType,
-      cdosLength, hideControls, classifications, hasErrored, pageNumber, isCDOD30 } = this.props;
-
+    const { bidderPortfolio, bidderPortfolioExtraData, bidderPortfolioFilteredSeasonDataSuccess, pageSize, showListView, isLoading, viewType,
+      cdosLength, hideControls, classifications, hasErrored, pageNumber, isCDOD30, isSeason, isClient, setEditClassification } = this.props;
+    console.log('bidderPortfolioFilteredSeasonDataSuccess BidConatiner', bidderPortfolioFilteredSeasonDataSuccess);
+    console.log('bidderPortfolio BidConatiner', bidderPortfolio);
```
Comment (fimbriano) [inline src/Components/BidderPortfolio/BidderPortfolioContainer/BidderPortfolioContainer.jsx:29]: remove logs
```suggestion
```

File: src/Constants/Sort.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { filter } from 'lodash';
-import { checkFlag } from 'flags';
+// import { checkFlag } from 'flags';
```
Comment (fimbriano) [inline src/Constants/Sort.js:2]:
```suggestion
import { checkFlag } from 'flags';
```

File: src/Constants/Sort.js [hunk 3]
```diff
@@ -108,7 +108,9 @@ export const BID_PORTFOLIO_FILTERS = {
 };
 
 BID_PORTFOLIO_FILTERS.defaultSort = BID_PORTFOLIO_FILTERS.options[0].value;
-const showCDOD30 = checkFlag('flags.CDOD30');
+// const showCDOD30 = checkFlag('flags.CDOD30');
+const showCDOD30 = true;
```
Comment (fimbriano) [inline src/Constants/Sort.js:112]: add FF back
```suggestion
const showCDOD30 = checkFlag('flags.CDOD30');
```

File: src/Containers/BidderPortfolio/BidderPortfolio.jsx [hunk 4]
```diff
@@ -132,20 +132,22 @@ class BidderPortfolio extends Component {
...
       bidderPortfolioCounts, bidderPortfolioCountsIsLoading, availableBiddersIdsLoading,
       bidderPortfolioCountsHasErrored, cdos, bidderPortfolioCDOsIsLoading,
       updatePagination, bidderPortfolioPagination, viewType,
       classifications, classificationsIsLoading, classificationsHasErrored } = this.props;
     const { hasHandshake, ordering, bidderIdsHasLoaded } = this.state;
     const isLoading = (bidderPortfolioCDOsIsLoading || bidderPortfolioIsLoading
       || (availableBiddersIdsLoading && !bidderIdsHasLoaded)) && cdos.length;
-
+    console.log('bidderPortfolioSeasonSearchFetchDataSuccess BidPortfolio', bidderPortfolioFilteredSeasonDataSuccess);
+    console.log('bidderPortfolio BidPortfolio', bidderPortfolio);
```
Comment (fimbriano) [inline src/Containers/BidderPortfolio/BidderPortfolio.jsx:144]: remove logs
```suggestion
```

File: src/actions/bidderPortfolio.js [hunk 5]
```diff
@@ -480,14 +541,15 @@ export function bidderPortfolioFetchData(query = {}, isCDO = false) {
...
     if (ids.length) {
-      query$.hru_id__in = ids.join();
-    }
-    if (isArray(seasons) && seasons.length) {
-      query$.bid_seasons = join(seasons, ',');
+      query$.hru_id = ids.join();
     }
+    // if (isArray(seasons) && seasons.length && isSeasons) {
+    //   query$.bid_seasons = join(seasons, ',');
+    // }
```
Comment (fimbriano) [inline src/actions/bidderPortfolio.js:552]: whats happening with the season here?

File: src/actions/bidderPortfolio.js [hunk 6]
```diff
@@ -539,6 +601,67 @@ export function bidderPortfolioFetchData(query = {}, isCDO = false) {
...
+    const endpoint = '/fsbid/client/';
+    const q = `${endpoint}?${query$$}`;
+    if (ids.length) {
+      api().get(q, {
+        cancelToken: new CancelToken((c) => {
+          cancelPortfolio = c;
+        }),
+      })
+        .then(({ data }) => {
+          console.log('BPSFD', data);
```
Comment (fimbriano) [inline src/actions/bidderPortfolio.js:639]: remove log
```suggestion
```

**Review by fimbriano — CHANGES_REQUESTED** [review]

File: src/actions/bidderPortfolio.js (hunk 5 above)
Comment (RamelHenderson) [reply src/actions/bidderPortfolio.js:552]: I don't know much because this was Charles's work 😮‍💨

Comment (fimbriano) [reply src/actions/bidderPortfolio.js:552]: hmm should we find out if this is needed then? lol

File: src/Components/BidderPortfolio/BidControls/BidControls.jsx [hunk 7]
```diff
@@ -311,16 +326,16 @@ class BidControls extends Component {
...
+          {/* { showCDOD30 && */}
+          <div className="jc-toggle-container">
+            <ToggleButton
+              labelTextRight={!isCDOD30 ? 'Toggle CDO D3.0 View' : 'Toggle Client Search View'}
+              onChange={this.onCDOToggleChange}
+              checked={isCDOD30}
+              onColor="#0071BC"
+            />
+          </div>
+          {/* } */}
```
Comment (fimbriano) [inline src/Components/BidderPortfolio/BidControls/BidControls.jsx:338]:
```suggestion
          { showCDOD30 && 
          <div className="jc-toggle-container">
            <ToggleButton
              labelTextRight={!isCDOD30 ? 'Toggle CDO D3.0 View' : 'Toggle Client Search View'}
              onChange={this.onCDOToggleChange}
              checked={isCDOD30}
              onColor="#0071BC"
            />
          </div>
         }
```

**Review by fimbriano — APPROVED** [review]

**Pull request merged**